cmd/frontend/backend/repos.go

```diff
@@ -245,3 +245,15 @@ func (s *repos) GetInventory(ctx context.Context, repo *types.Repo, commitID api
 	}
 	return &inv, nil
 }
+
+func (s *repos) Delete(ctx context.Context, name api.RepoName) (err error) {
```

Comment:
I'd rename this here too.

```suggestion
func (s *repos) DeleteRepositoryFromDisk(ctx context.Context, name api.RepoName) (err error) {
```